lib/ansible/modules/packaging/language/pip.py

```diff
@@ -503,12 +570,24 @@ def main():
         # Automatically apply -e option to extra_args when source is a VCS url. VCS
         # includes those beginning with svn+, git+, hg+ or bzr+
         has_vcs = False
+
         if name:
             for pkg in name:
                 if bool(pkg and re.match(r'(svn|git|hg|bzr)\+', pkg)):
                     has_vcs = True
                     break
 
+            distributions = _get_distributions_from_names(module, name)
+            # check invalid combination of arguments
+            if version is not None:
+                if len(distributions) > 1:
+                    module.fail_json(msg="Version arg is not available for installing multi-packages.")
+                else:
+                    if distributions[0].has_version_specifier:
+                        module.fail_json(msg="Version arg is not available if version info is provided in name.")
```

Comment:
I'd rephrase this into `'version' argument conflicts with version specifier provided along with a package name`.